new_twitter.css

```diff
@@ -0,0 +1,243 @@
+*{
+    margin: 0;
+}
+.container{
+    display:grid;
+    height: 100vh;
+    grid-template:
+    'left-sidebar main right-sidebar'/300px 1fr 300px
+}
+.logo{
```

Comment:
You should add an empty line after every CSS block to give some breathing space, it makes it a little easier to read / maintain